toplevel/topdirs.ml

```diff
@@ -447,6 +446,13 @@ let _ = add_directive "ppxs_clear"
              (external and in-process)";
     }
 
+let _ = add_directive "remove_in_process_ppxs"
+    (Directive_none Pparse.clear_in_process_ppx)
+    {
+      section = section_options;
+      doc = "Forget all added in process PPX preprocessors";
+    }
```

Comment:
@whitequark Do you propose to use path as a name for in-process ppx? And `#remove_in_process_ppx` will take this path to decide which ppx to remove?